src/BPEssentials/Commands/Moderation/Jail.cs

```diff
@@ -0,0 +1,39 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BrokeProtocol.Entities;
+using BrokeProtocol.Required;
+using BrokeProtocol.Utility;
+using BrokeProtocol.Utility.Jobs;
+using BrokeProtocol.Utility.Networking;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using UnityEngine;
+
+namespace BPEssentials.Commands
+{
+	public class Jail : Command
+	{
+		public void Invoke(ShPlayer player, ShPlayer target, float timeInSeconds)
+		{
+			ShJail jail = target.manager.jails.FirstOrDefault();
+			if (jail)
+			{
+				if (target.IsDead || target.job is Prisoner)
+				{
+					return;
+				}
+				Transform getPositionT = jail.GetPositionT;
```

Comment:
Use `var` wherever possible.
```suggestion
				var getPositionT = jail.GetPositionT;
```